Crashes while setting null auth cookies

**Summary:** 
App crashes while setting null auth cookies in CustomMwApi
**System logs:**

```
STACK_TRACE=java.lang.RuntimeException: An error occurred while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:353)
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
at java.util.concurrent.FutureTask.run(FutureTask.java:271)
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:245)

at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
at java.lang.Thread.run(Thread.java:764)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String[] java.lang.String.split(java.lang.String)' on a null object reference
at fr.free.nrw.commons.mwapi.CustomMwApi.setAuthCookie(CustomMwApi.java:74)
at fr.free.nrw.commons.mwapi.ApacheHttpClientMediaWikiApi.setAuthCookie(ApacheHttpClientMediaWikiApi.java:199)
at fr.free.nrw.commons.delete.DeleteTask.doInBackground(DeleteTask.java:74)
at fr.free.nrw.commons.delete.DeleteTask.doInBackground(DeleteTask.java:28)
at android.os.AsyncTask$2.call(AsyncTask.java:333)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
... 4 more
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String[] java.lang.String.split(java.lang.String)' on a null object reference
at fr.free.nrw.commons.mwapi.CustomMwApi.setAuthCookie(CustomMwApi.java:74)
at fr.free.nrw.commons.mwapi.ApacheHttpClientMediaWikiApi.setAuthCookie(ApacheHttpClientMediaWikiApi.java:199)
at fr.free.nrw.commons.delete.DeleteTask.doInBackground(DeleteTask.java:74)
at fr.free.nrw.commons.delete.DeleteTask.doInBackground(DeleteTask.java:28)
at android.os.AsyncTask$2.call(AsyncTask.java:333)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:245)

at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
at java.lang.Thread.run(Thread.java:764)

```

**Device and Android version:** 

What make and model device (e.g., Samsung J7) did you encounter this on?
ANDROID_VERSION=8.1.0
APP_VERSION_NAME=2.10.2
BRAND=vivo
PHONE_MODEL=vivo 1718
 
**Commons app version:** 
APP_VERSION_NAME=2.10.2

**Would you like to work on the issue?**
Yes

--- a/app/src/main/java/fr/free/nrw/commons/mwapi/CustomMwApi.java
+++ b/app/src/main/java/fr/free/nrw/commons/mwapi/CustomMwApi.java
@@ -69,6 +69,10 @@ public class CustomMwApi {
     }
 
     public void setAuthCookie(String authCookie) {
+        if (authCookie == null) {//If the authCookie is null, no need to proceed
+            return;
+        }
+
         this.authCookie = authCookie;
         this.isLoggedIn = true;
         String[] cookies = authCookie.split(";");

